### DISCUSSION TITLE
Ideas concerning Scenario selection

### DISCUSSION THREAD

**Discussion by Superjustinbros** [OpenRCT2/OpenRCT2#15677] [Feature requests]
**OS:** Windows 7 64 bit
**Version:** 0.04

Today I decided to update OpenRCT2, and after taking a gander at the revised Scenario Selection menu, I wanted to provide some ideas for it.
- ~~On my computer, I have many incarnations of the basic RCT1 scenarios (like three Forest Frontiers and three Dynamite Dunes) instead of just one. All variations of the scenarios usually have differences, like the number of years available/guests required to complete the objective. In this case, I'd recommend after selecting the scenario to be provided with "variations" that contain the same park if the menu names match perfectly. Or a way to remove the duplicates or not display them.~~
- ~~Listing filenames and their source addresses, since I have multiple downloads of RCT2 downloaded and it doesn't otherwise state which one it's reading from unless I dig deep into the options.~~
- A search feature to search for scenario names would help, considering how many scenarios I have downloaded.

**Comment by AaronVanGeffen** [comment]
> Listing filenames and their source addresses, since I have multiple downloads of RCT2 downloaded and it doesn't otherwise state which one it's reading from unless I dig deep into the options.

If you enable debugging tools in the options window, it will show the file names on mouseover.

**Comment by Gymnasiast** [comment]
Point 1 has been addressed as well.
